packages/identity/src/errors/print.ts

```diff
@@ -1,29 +1,26 @@
-// @flow
-
 // Use a RegExp since IE11 does not implement Function.prototype.name
 const constructorNameRegExp = /^.*(?:function|class) +([^( ]+).*$/;
 
-const getConstructorName = (obj: Object) => {
+const getConstructorName = (obj: Record<string, any>) => {
```

Comment:
```suggestion
const getConstructorName = (obj: Object) => {
```

`obj` can be anything.